person.rb

```diff
@@ -4,20 +4,25 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rental
```

Comment:
- [ ] [REQUIRED] Good job so far! Please implement `Getters` and `Setters` for `@rentals` should be present in `attr_accessor` 🙏🏼 

```ruby
attr_accessor :name, :age, :rentals
```